library/Stratosphere/Types.hs

```diff
@@ -1,19 +1,170 @@
-{-# LANGUAGE DeriveAnyClass #-}
-{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE DeriveAnyClass    #-}
+{-# LANGUAGE DeriveGeneric     #-}
 {-# LANGUAGE OverloadedStrings #-}
 
 -- | Module for hand-written types that are used in generated modules.
 
 module Stratosphere.Types
-  ( CannedACL (..)
+  ( EnabledState (..)
+  , AuthorizationType (..)
+  , HttpMethod (..)
+  , LoggingLevel (..)
+  , ApiBackendType (..)
+  , Period (..)
+  , AttributeType (..)
+  , KeyType (..)
+  , ProjectionType (..)
+  , StreamViewType (..)
+  , SNSProtocol (..)
+  , Runtime (..)
+  , CannedACL (..)
   , KinesisFirehoseS3CompressionFormat(..)
   , KinesisFirehoseElasticsearchS3BackupMode(..)
   , KinesisFirehoseNoEncryptionConfig(..)
   ) where
 
-import Data.Aeson
-import Data.Text (unpack)
-import GHC.Generics
+import           Data.Aeson
+import           Data.Text    (unpack)
+import           GHC.Generics
+
+
+data EnabledState
```

Comment:
Yes, this is exactly what i was thinking - should be ok for now. Although, we should eventually come up with a good naming convention, since changing these down the road would break the code that depends on the library.
Speaking of naming, there are also visible inconsistencies in resource naming, i think it makes sense to prefix those with the AWS service name, but some of resource names are not currently prefixed.